src/components/ContentContainer.tsx

```diff
@@ -1,9 +1,10 @@
+import { Grid } from '@mui/material';
 import React, { FunctionComponent, ReactElement } from 'react';
-import Grid from '@mui/material/Unstable_Grid2';
+// import Grid from '@mui/material/Unstable_Grid2';
 import TopLogo from '../assets/images/top_logo.png';
 
 interface Props {
-  children: ReactElement[] | ReactElement;
+  children: ReactElement[] | ReactComponent[];
```

Comment:
@fargusplumdoodle should this be ReactComponent[] or stay ReactElement?

Reply:
Yes this should probable be the way it was before, I don't know what putting it as a `ReactComponent` will do here

Reply:
Maybe stick with Grid 2 if it doesn't cause issues?